### PR TITLE
Adding MonadA

### PR DESCRIPTION
**Pull request by Katochimoto**

### CODE REVIEW

File: registry.json
Comment (Bez625) [inline registry.json:3614]: Please can you add a user TG handle? This must be the handle of a user and not a group or channel.

File: registry.json
Comment (Bez625) [inline registry.json:3606]: I checked your pool and it says your pool's pledge has not been met. I'm going to request you fix this before I approve if only to raise your awareness of the situation...

**Review by Bez625 — CHANGES_REQUESTED** [review]

File: registry.json
Comment (Katochimoto) [reply registry.json:3662]: Had to wait for the end of an epoch

File: registry.json
Comment (Katochimoto) [reply registry.json]: Now should be okay, thank you

**Pull request merged**
